.travis.yml

```diff
@@ -6,6 +6,9 @@ python:
   - "3.4"
   - "2.7"
 
+node_js:
+  - "8"
```

Comment:
this should not be needed (we have added similar checks in other modules without adding this), try to remove it